CHANGELOG.md

```diff
@@ -1,11 +1,13 @@
 Changes by Version
 ==================
 
-2.22.2 (unreleased)
+2.23.0 (2020-04-22)
 -------------------
 
-- Nothing yet.
-
+- Add the ability to log all span interactions and a debug log level([#502](https://github.com/jaegertracing/jaeger-client-go/pull/502), [#503](https://github.com/jaegertracing/jaeger-client-go/pull/503), [#504](https://github.com/jaegertracing/jaeger-client-go/pull/504)) -- Prithvi Raj
+- internal: Use port 6831 for xdock ([#500](https://github.com/jaegertracing/jaeger-client-go/pull/500)) -- Pavol Loffay
```

Comment:
this doesn't need to be in the release notes, it's an internal testing fix

Reply:
removed that line